discovery-provider/scripts/start.sh

```diff
@@ -110,7 +110,6 @@ if [ "$audius_db_run_migrations" != false ]; then
     echo "Finished running migrations"
 fi
 
-
 # start es-indexer
 if [[ "$audius_elasticsearch_url" ]] && [[ "$audius_elasticsearch_run_indexer" ]]; then
```

Comment:
in this new world, i don't think there's ES implemented yet right? not sure what the es indexer will do.

also noticed in playing with client, we don't have AAO either.